README.md

```diff
@@ -787,32 +812,32 @@ Now the prover creates witness for the commitment `Statement` using the randomne
 const committeds = [blinding];
```

Comment:
There are helpers for creating `Witness` object easy for blind signatures. See `getBBSWitnessForBlindSigRequest`, `getBBSPlusWitnessForBlindSigRequest`, `getPSWitnessesForBlindSigRequest`. See usage [here](https://github.com/docknetwork/crypto-wasm-ts/blob/master/tests/composite-proofs/social-kyc.spec.ts#L112).